lib/actions/device.coffee

```diff
@@ -384,8 +384,8 @@ exports.init =
 				if process.env.DEBUG
 					console.log(results.config)
 
-				bar = new visuals.widgets.Progress('Downloading Device OS')
-				spinner = new visuals.widgets.Spinner('Downloading Device OS (size unknown)')
+				bar = new visuals.Progress('Downloading Device OS')
```

Comment:
Yeah, actually i was hoping you gonna ask that. I could't test it due to the fact that i don't have admin privileges yet.